lib/models/factories.js

```diff
@@ -147,7 +147,7 @@ function deviceFactory(overrides = {}) {
 
 async function locationDBFactory(db, overrides = {}) {
   const device = await db.createDevice(
-    DEVICE_TYPE.DEVICE_SENSOR,
+    DEVICE_TYPE.DEVICE_SENSOR_SINGLESTALL,
```

Comment:
is it wise to assume that a locationDBFactory will be single stall? Maybe we can add DEVICE_TYPE as an override like every other variable in this object.

Reply:
I was thinking to change the test cases accordingly with everything to be single stall. However adding an override will also be a good idea. We should discuss this